README.md

```diff
@@ -1,66 +1,120 @@
+* PostgreSQL (+ development headers)
+
+Cooking Instructions:
+---  
+1.  **Set up the environment.**
+
+    	git clone http://github.com/kancolletool/kcsrv.git kcsrv
+    	cd kcsrv
+    	virtualenv -p /usr/bin/python3 .
+    	source ./bin/activate
+        
+    Now, enter the project's Virtual Environment. This contraption allows you to install packages locally, without conflicting with system packages. It's activated per-terminal, and must be activated before interacting with the project in any way:
+  
+        # Not a typo; `.` is an alias for `source`  
+        . bin/activate
+        
+    If you wish to exit it again, simply run:
+
+        deactivate
+        
+1. **Install dependencies.**  
+    Inside the virtualenv, simply run:
+
+    	pip install -r requirements.txt
+
+    If you ever get issues with things not being found, try re-running this and seeing what happens.
+    
+1. **Set up the database***  
+    First, make sure you have a database user for the account you plan on running the server as. Postgres can automatically authorize as the user a process is running as, which means there's no need to put passwords in configuration files.
 
+        sudo -u postgres createuser --superuser $USER
+        
+    Then, create a database for kcsrv to use:
+        sudo -u postgres createdb kcsrv  
 
-1.  **Create the database.**  
-    Repeat this to upgrade it later.
+    Change postgres to what user the server installs as. On Debian/Ubuntu it's postgres.
+    And fill it with data - repeat this after every upgrade:
+    
+        ./manage.py db upgrade   # Run migrations
+        ./manage.py db update_db # Imports ship data into the database
+        
+    First Boot? You'll need to set up a few things.
     
-        ./manage.py db upgrade
         ./manage.py setup
-        ./manage.py update_db
+
+1. **Download the DMM assets.**  
+    This will take a while... you may want to re-run it after game patches, to stay up-to-date. At this point, grab a coffee.
+
+    Please note that this won't pull new ships automatically, you'll need to feed the server a new api_start2.json every time DMM introduces more ships to collect. Gotta ship 'em all!
+
+        ./manage.py dlassets
 
 1.  **Create an account for yourself.**  
     Be sure to give yourself the `admin` and `staff` roles!
-    
-        ./manage.py user create
+    Depending on your terminal, you may need to put the email address in quotation marks.
 
+        ./manage.py user create <username> <email>
+        # Example: ./manage.py user create MyAdmiralName "you@localhost"
 
 1.  **Run the development server.**  
     It will run on port 5000 by default.
-    
+
         ./kcsrv.py
+
+    Since the game is hardcoded to use port 80 instead of port 5000, you'll likely get stuck at a floating ship screen. This is not a bug - it's poor DMM design. You'll need to run a SSH tunnel or use a good webserver like nginx to be a reverse proxy. More details on that later.
```

Comment:
Better wording:

1. **Connect to the game**
    
    Since the game will always connect on Port 80 (blame Flash), you need to proxy the dev server to your local Port 80:
    
        sudo ssh localhost -L 80:localhost:5000